content/hardware/08.edu/solution-and-kits/alvik/tutorials/user-manual/user-manual.md

```diff
@@ -78,7 +87,7 @@ At the back-right side of Alvik there is the main switch of the robot. When ON t
 
 ![Notification to switch on](assets/message-switch-on.png)
 
-#### Battery
+### Battery
 
 The battery is a rechargeable Li-ion 18650. It is located in the bottom part of Alvik, to access it you need to remove one Phillip's screw and take out the plastic holder.

```

Comment:
we need to add here information about how to proceed when replacing the battery. Users will need to:
- Detach existing battery
- Attaching a new battery
- Plug NanoESP32 (in case they are using other controller or using the robot in a different manner)
- Plug the NanoESP32 to the computer
Unless users follow these steps, the robot won't work after a battery replacement, these steps are needed to reset the BMS after a battery change. If users don't do this, the BMS is "off" and the robot won't turn on

Reply:
Added